prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -75,6 +76,7 @@ def load(self) -> None:
 
 class DeployableOlasEmbeddingOAAgent(DeployableTraderAgentER):
     agent: OlasAgent
+    bet_on_n_markets_per_run = 20
```

Comment:
```
input $2.5 / 1M tokens
output $10.00 / 1M tokens
```

Reply:
maybe a side-issue, but I wonder if we should retire this guy. Not sure what benefit we have from keeping it deployed - we track the actual olas agents on dune, and the code its using to make predictions is from the prediction_prophet repo, so over time it will get out-of-date with what the olas agents are doing anyway.

Reply:
i thought we can just leave him as long as he doesn't die out, but the name is not great that's true